receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -134,8 +136,8 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 				// See https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/gitproviderreceiver/internal/scraper/githubscraper/README.md#github-limitations
 				// for more information as to why `BehindBy` and `AheadBy` are
```

Comment:
It looks like there are some comments referring to old metric names, they should probably be updated, as well as in the githubscraper README.